frontend/src/utils/og.ts

```diff
@@ -1,5 +1,5 @@
 import type { MetaInfo } from "vue-meta"
-import type { MetaPropertyName } from "vue-meta/types/vue-meta"
+import type { MetaPropertyProperty } from "vue-meta/types/vue-meta"
```

Comment:
These types are imported from the `vue-meta` package and must use the name set by that package. Please revert this change.